tests/specs/lemmas.k

```diff
@@ -204,33 +261,24 @@ module LEMMAS-JAVA [symbolic, kast]
     rule bool2Word(A) =/=K 0 => A          [simplification]
     rule bool2Word(A) =/=K 1 => notBool(A) [simplification]
 
-    rule notBool  (X ==K 0) => X ==K 1 requires #rangeBool(X) [simplification]
-    rule notBool  (X ==K 1) => X ==K 0 requires #rangeBool(X) [simplification]
-    rule bool2Word(X ==K 1) => X       requires #rangeBool(X) [simplification]
-
-    rule B ==K false => notBool B [simplification]
-
-    rule 0 <=Int X &Int Y             => true requires 0 <=Int X andBool X <Int pow256 andBool 0 <=Int Y andBool Y <Int pow256 [simplification]
-    rule         X &Int Y <Int pow256 => true requires 0 <=Int X andBool X <Int pow256 andBool 0 <=Int Y andBool Y <Int pow256 [simplification]
+    rule notBool  (X ==K 0) => X ==K 1  requires #rangeBool(X) [simplification]
+    rule notBool  (X ==K 1) => X ==K 0  requires #rangeBool(X) [simplification]
+    rule bool2Word(X ==K 1) => X        requires #rangeBool(X) [simplification]
 
-    rule 0 <=Int #asWord(_WS)         => true [simplification]
-    rule #asWord(_WS) <Int pow256     => true [simplification]
+    rule 0 <=Int #asWord(_WS)             => true [simplification]
+    rule         #asWord(_WS) <Int pow256 => true [simplification]
```

Comment:
Are these needed? I don't think they're true. Take for example `WS ==WordStack -256 : .WordStack`.

Reply:
Ah wait, nevermind. `#asWord` injects a `chop`.